client/verta/verta/environment/_environment.py

```diff
@@ -29,6 +30,20 @@ def __init__(self, env_vars, autocapture):
             self._capture_env_vars(env_vars)
         if autocapture:
             self._capture_cmd_line_args()
+        if apt_packages:
+            self.apt_packages = apt_packages
+
+    @property
+    def apt_packages(self):
```

Comment:
I decided to hold off on overhauling this class for now and continue to store the data in the protobuf message